website/content/docs/commands/index.mdx

```diff
@@ -172,6 +172,10 @@ Or specify the contents of a file as a value:
 $ vault kv put secret/password value=@data.txt
 ```
 
+Note that if an argument is supplied in a @key=value format, Vault will treat that as a 
+kv pair with the key being `@key`, not a file called `key=value`. This also means that Vault
```

Comment:
Technically we do support filenames with `=` in them in some cases right?

Reply:
hmm no i don't think so. We first split the input by `=` here https://github.com/hashicorp/go-secure-stdlib/blob/main/kv-builder/builder.go#L52, and if len(parts) == 1 i.e the input has no `=`, then we try to read it as a file here https://github.com/hashicorp/go-secure-stdlib/blob/main/kv-builder/builder.go#L70

Reply:
Ah, my mistake then.